IoTuring/Warehouse/Deployments/HomeAssistantWarehouse/entities.yaml

```diff
@@ -124,3 +124,8 @@ TerminalButton:
   name: Terminal Button
   icon: mdi:console
   custom_type: button
+Brightness:
```

Comment:
You can create a light entity, it would fit better:
- https://developers.home-assistant.io/docs/core/entity/light
- https://www.home-assistant.io/integrations/light.mqtt/

Reply:
Yes that would integrate really well

Reply:
I changed it to `custom_type : light`.
The EntityType changes, but i dont get the interface to change.
![image](https://github.com/user-attachments/assets/34cc7a7a-b891-44dc-87df-a9fd630417de)
![image](https://github.com/user-attachments/assets/f68a0eb8-4bff-4121-8be1-72e512d7b159)

Reply:
> You can create a light entity, it would fit better:
> 
>     * https://developers.home-assistant.io/docs/core/entity/light
> 
>     * https://www.home-assistant.io/integrations/light.mqtt/

What do you mean by "create"?

I agree with it fitting better than 'number' as EntityType. Although the LightEntity is pretty powerfull and DisplayBacklights will not be able to gain from those capabilities like RGB, Hue, Saturation controls.
Also LightEntity gets a binary state as the default control, which is not fitting for displaybacklights. 
![image](https://github.com/user-attachments/assets/61f39ffa-26c4-49cd-899b-e2e51c916312)